### PR TITLE
Docs edits

### PR DESCRIPTION
**Pull request by jwithington**
## Description

Some grammar, tense, and passive voice edits for a few docs pages. 

## Related Issues

Related to #18284 

Thanks!

### CODE REVIEW

File: docs/docs/adding-an-rss-feed.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ npm install --save gatsby-plugin-feed
 
 ## How to use [gatsby-plugin-feed](/packages/gatsby-plugin-feed/)
 
-Once installation is complete, you can now add this plugin to your site's config file, like so:
+Once you complete installation, you can add the plugin to your site's config file, like so:
```
Comment (KyleAMathews) [inline docs/docs/adding-an-rss-feed.md]: This is an npm installation? So it'd be "once NPM finishes installation"

Comment (jwithington) [reply docs/docs/adding-an-rss-feed.md]: Good question! I edited for passive voice, not for content, but can definitely change that portion!

Comment (jwithington) [reply docs/docs/adding-an-rss-feed.md]: Edited to make it even more concise:

> At this point, you can add the plugin to your site's config file, like so:

**Comment by marcysutton** [comment]
Hey! Thanks so much for opening a pull request!

We really appreciate you sending this over, but the change you’ve proposed is not going to be accepted because it doesn't meaningfully improve docs, fix a bug, or otherwise improve functionality.

We absolutely want to have you as a contributor, so please take a look at [our open issues](https://github.com/gatsbyjs/gatsby/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for ideas, and please reach out to us [on Twitter at @gatsbyjs](https://twitter.com/gatsbyjs) with questions. We do have the Hacktoberfest issue open you've referenced, which includes specific requests such as fixing pronouns or heading levels.
 
Thanks again, and we look forward to seeing more PRs from you in the future! 💪💜

**Pull request closed without merging**